docs/reference/action_list.rst

```diff
@@ -50,6 +44,33 @@ For more detailed information about routes, see :doc:`routing`::
         }
     }
 
+Custom route
+^^^^^^^^^^^^
+
+Default route for a link is `show` (for `FieldDescriptionInterface::TYPE_MANY_TO_ONE` and `FieldDescriptionInterface::TYPE_ONE_TO_ONE`).
+Using this, the route can be customized as follows::
+
+    namespace Sonata\MediaBundle\Admin;
```

Comment:
```suggestion
    namespace App\Admin;
```
If these are examples for users to use, I would use `App` namespace (in all examples) to make it clear.

Reply:
Done!